client/src/js/templates/dc/grid.html

```diff
@@ -22,7 +22,7 @@ <h1>
     <ul class="clearfix single">
         <% if (BLSAMPLEID) { %><li class="sample"><span class="wrap">Sample: <a href="/samples/sid/<%-BLSAMPLEID%>"><%-SAMPLE%></a></span></li><% } %>
         <li>&Omega; Start: <%-AXISSTART%>&deg;</li>
-        <% if ((KAPPA && KAPPA !=0) || (PHI && PHI != 0)) { %><li>&kappa;: <%-KAPPA%>&deg; &phi;: <%-PHI%>&deg;</li><% } %>
+        <% if ((KAPPA && KAPPA !=0) || (PHI && PHI != 0) || (CHISTART && CHISTART !=0)) { %><li><% if (KAPPA != null) { %>&kappa;: <%-KAPPA == null ? 0 : KAPPA%>&deg;<% } %> <% if (PHI != null) { %>&phi;: <%-PHI == null ? 0 : PHI%>&deg;<% } %> <% if (CHISTART != null) { %>&chi;: <%-CHISTART == null ? 0 : CHISTART%>&deg;<% } %></li><% } %>
```

Comment:
This line: <% if (KAPPA != null) { %>&kappa;: <%-KAPPA == null ? 0 : KAPPA%>&deg;<% } %> seems to include a redundant check. Would this work?
<% if (KAPPA != null) { %>&kappa;: <%-KAPPA%>&deg;<% } %>
Same for PHI and CHISTART?

Reply:
Sounds sensible, need to make it consistent with dc.html too